assimilation_code/programs/obs_selection/obs_selection.rst

```diff
@@ -1,3 +1,5 @@
+.. _obs_selection:
```

Comment:
Also include 

`.. index:: obs selection `

so it is easily searchable like was done for the time manager?

Reply:
yup

Reply:
added in [0e0ba95](https://github.com/NCAR/DART/pull/832/commits/0e0ba9572c3a2beba9901104b736c390e3ccc2d4)